_i18n/template/resources/developer-guides/wallet-rpc.md

```diff
@@ -204,15 +257,18 @@ Inputs:
 * *destinations* - array of destinations to receive XMR:
   * *amount* - unsigned int; Amount to send to each destination, in atomic units.
   * *address* - string; Destination public address.
-* *fee* - unsigned int; Ignored, will be automatically calculated.
+* *account_index* - unsigned int; Transfer from this account.
+* *subaddr_indices* - array of unsigned int; Transfer from this set of subaddresses.
 * *mixin* - unsigned int; Number of outpouts from the blockchain to mix with (0 means no mixing).
+* *ring_size* - unsigned int; Amount to adjust mixin. Accepted values: 0 (default mixin, currently 6), n >= default (set mixin to n).
```

Comment:
Incorrect, it sets ring size to n (mixin+1)

Reply:
Ok, was a bit confused. Makes sense now, will update.